Event/SendinblueListSubscriber.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\FormBundle\Event;
+
+use SendinBlue\Client\Api\ContactsApi;
+use SendinBlue\Client\Configuration;
+use SendinBlue\Client\Model\CreateDoiContact;
+use Sulu\Bundle\FormBundle\Entity\Dynamic;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+class SendinblueListSubscriber implements EventSubscriberInterface
+{
+    /**
+     * @var RequestStack
+     */
+    private $requestStack;
+
+    /**
+     * @var ContactsApi
+     */
+    private $contactsApi;
+
+    public function __construct(RequestStack $requestStack, ?string $apiKey)
+    {
+        $this->requestStack = $requestStack;
+
+        $config = new Configuration();
+        $config->setApiKey('api-key', $apiKey);
+
+        $this->contactsApi = new ContactsApi(null, $config);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public static function getSubscribedEvents()
+    {
+        return [
+            FormSavePostEvent::NAME => 'listSubscribe',
+        ];
+    }
+
+    public function listSubscribe(FormSavePostEvent $event): void
+    {
+        $dynamic = $event->getData();
+        $request = $this->requestStack->getCurrentRequest();
+
+        if (!$dynamic instanceof Dynamic) {
+            return;
+        }
+
+        if (!$request) {
+            return;
+        }
+
+        $form = $dynamic->getForm()->serializeForLocale($dynamic->getLocale(), $dynamic);
+
+        $email = '';
+        $firstName = '';
+        $lastName = '';
+        $redirectionUrl = $request->getUriForPath('');
```

Comment:
@luca-rath should we add`?subscribe=true`. So in the frontend a message can be shown?

Reply:
```suggestion
        $redirectionUrl = $request->getUriForPath('') . '?subscribe=true';
```